lib/bundler/cli.rb

```diff
@@ -73,6 +74,32 @@ def self.handle_no_command_error(command, has_namespace = $thor_runner)
       Kernel.exec(command_path, *ARGV[1..-1])
     end
 
+    
+
+    Bundler.plugin("1").manager.commands.all.each do |command,proc|
```

Comment:
It is creating a method for every registered command. That's how Thor defines sub commands. I did look for ways to do it dynamically but couldn't find one that worked. There's probably a better way to do it.